packages/experimental/Menu/src/MenuItem/MenuItemTokens.ts

```diff
@@ -0,0 +1,29 @@
+import { FontWeightValue, Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { MenuItemTokens } from './MenuItem.types';
+
+export const defaultMenuItemTokens: TokenSettings<MenuItemTokens, Theme> = (t: Theme): MenuItemTokens => ({
+  backgroundColor: t.colors.neutralBackground1,
```

Comment:
Are these "defaults" just the web tokens?

Reply:
Yeah, the ones in this file are from web.